src/components/Data/Indexes/Page.vue

```diff
@@ -0,0 +1,187 @@
+<template>
+  <b-container class="IndexesPage">
+    <headline>
+      Indexes
+      <b-button
+        class="float-right mt-3"
+        data-cy="IndexesPage-createBtn"
+        v-if="canCreateIndex()"
+        variant="primary"
+        :title="
+          !canCreateIndex() ? `Your rights disallow you to create indexes` : ''
+        "
+        @click.prevent="openCreateModal"
+      >
+        <i class="fa fa-plus-circle left" />
+        Create an index
+      </b-button>
+    </headline>
+
+    <list-not-allowed v-if="!canSearchIndex()" />
+    <template v-else>
+      <b-row class="mb-3">
+        <b-col sm="8" class="text-secondary">
```

Comment:
this div is not vertically aligned with the input filter on the right.
```suggestion
        <b-col sm="8" class="text-secondary pt-2">
```